File: AspNetMvcUnleashed/0672329980_source/Chapter09/VB/MvcApplication1/MvcApplication1/Controllers/SortController.vb
Public Class SortController
    Inherits System.Web.Mvc.Controller

    Public Function Index(ByVal values As String) As String
        Dim brokenValues = values.Split("/"c)
        Array.Sort(brokenValues)
        Return String.Join(", ", brokenValues)
    End Function

End Class

File: AspNetMvcUnleashed/0672329980_source/Chapter09/CS/MvcApplication1/MvcApplication1/Controllers/SortController.cs
using System;
using System.Web.Mvc;

namespace MvcApplication1.Controllers
{
    public class SortController : Controller
    {

        public string Index(string values)
        {
            var brokenValues = values.Split('/');
            Array.Sort(brokenValues);
            return String.Join(", ", brokenValues);
        }


    }
}
